docker/build/devdeps.manylinux.Dockerfile

```diff
@@ -46,6 +46,7 @@ RUN dnf check-update && dnf install -y --nobest --setopt=install_weak_deps=False
 ADD ./scripts/install_prerequisites.sh /scripts/install_prerequisites.sh
 ENV BLAS_INSTALL_PREFIX=/usr/local/blas
 ENV OPENSSL_INSTALL_PREFIX=/usr/local/openssl
+ENV METIS_INSTALL_PREFIX=/opt/metis
```

Comment:
This will cause metis to be installed, but then likely not used in the build of the python wheels because METIS_DIR is not passed to cmake. We don't want to include metis in the wheels, but I would suggest to update the prerequisites script to simply not install metis unless METIS_INSTALL_PREFIX is defined and the needed library does not already exist. 
```suggestion
```